examples/camera-app/linux/src/clusters/camera-avstream-mgmt/camera-av-stream-manager.cpp

```diff
@@ -195,13 +195,19 @@ CHIP_ERROR CameraAVStreamManager::ValidateAudioStreamID(uint16_t audioStreamId)
 CHIP_ERROR CameraAVStreamManager::IsPrivacyModeActive(bool & isActive)
```

Comment:
Not part of this PR. But why do we have a single IsPrivacyModeActive()? We should have separate ones for SoftLivestreamPrivacy and RecordingPrivacy, isn't it?

Reply:
In all three scenarios, SolicitOffer, ProvideOffer and PushAV, we need same handling

1. If HardPrivacyModeOn is true:
Fail the command with the status code INVALID_IN_STATE, and end processing with no other side effects.

2. If SoftLivestreamPrivacyModeEnabled is true and StreamUsage is LiveView:
Fail the command with the status code INVALID_IN_STATE, and end processing with no other side effects.

3. If SoftRecordingPrivacyModeEnabled is true and StreamUsage is Recording or Analysis:
Fail the command with the status code INVALID_IN_STATE, and end processing with no other side effects.

IsPrivacyModeActive has already taken care of 2, 3, now, I am adding 1 to this check

Reply:
Let me land the current PR first, if we need to separate them, then we also need to separate the SoftLivestreamPrivacyModeEnabled and SoftRecordingPrivacyModeEnabled, I will use a separate PR to conduct this refactor, but I don't see the reason we should do this now.